dvc/path_info.py

```diff
@@ -57,6 +57,16 @@ def __repr__(self):
     def __fspath__(self):
         return pathlib.PurePath.__str__(self)
 
+    def __eq__(self, other):
+        if isinstance(other, str):
+            return self.fspath == os.path.abspath(other)
+        else:
+            return super().__eq__(other)
+
+    # pylint: disable=W0235
+    def __hash__(self):
+        return super().__hash__()
```

Comment:
Not needed?

Reply:
When we change `__eq__` , we also need to provide an implementation for `__hash__`.